content/SCALE/SCALEUIReference/SystemSettings/Services/SMBServicesScreen.md

```diff
@@ -46,10 +46,10 @@ Click **Save** or **Cancel** to close the configuration screen and return to the
-| **Log Level** | Record SMB service messages up to the specified log level from the dropdown list. Options are **None**, **Minimum**, **Normal**, **full** and **Debug**. By default, TrueNAS logs error and warning-level messages. We do not recommend using a log level above **Minimum** for production servers. |
+| **Use Debug** | Select to log more detailed information about SMB. By default, TrueNAS logs error and warning-level messages. We do not recommend enabling debug logging for production servers as it can generate large log files and impact performance. |
 | **Use Syslog Only** | Select to log authentication failures in <file>/var/log/messages</file> instead of the default <file>/var/log/samba4/log.smbd</file>. |
 | **Local Master** | Selected by default and determines if the system participates in a browser election. Leave cleared when the network contains an Active Directory or LDAP server or when Vista or Windows 7 machines are present. |
-| **Enable Apple SMB2/3 Protocol Extensions** | Select to allow MacOS to use these [protocol extensions](https://support.apple.com/en-us/HT210803) to improve the performance and behavioral characteristics of SMB shares. TrueNAS requires Apple SMB2/3 protocol extensions for Time Machine support. |
+| **Enable Apple SMB2/3 Protocol Extensions** | Select to allow MacOS to use these [protocol extensions](https://support.apple.com/en-us/HT210803) to improve the performance and behavioral characteristics of SMB shares. TrueNAS requires Apple SMB2/3 protocol extensions for Time Machine support and MacOS Media Share (Apple M&E workflows). This setting must be enabled before creating shares with the **Time Machine Share** or **MacOS Media Share** purpose options. |
```

Comment:
define what M&E is the first time it appears in the article. and if they call it M&E then put that in bold, or you can just say "media and entertainment"
```suggestion
| **Enable Apple SMB2/3 Protocol Extensions** | Select to allow MacOS to use these [protocol extensions](https://support.apple.com/en-us/HT210803) to improve the performance and behavioral characteristics of SMB shares. TrueNAS requires Apple SMB2/3 protocol extensions for Time Machine support and MacOS Media Share (Apple Media & Entertainment workflows). This setting must be enabled before creating shares with the **Time Machine Share** or **MacOS Media Share** purpose options. |
```